app/code/Magento/Quote/Test/Unit/Model/Quote/Address/Total/SubtotalTest.php

```diff
@@ -161,10 +165,13 @@ public function testCollect($price, $originalPrice, $itemHasParent, $expectedPri
         $shippingAssignmentMock->expects($this->exactly(2))->method('getShipping')->willReturn($shipping);
         $shippingAssignmentMock->expects($this->once())->method('getItems')->willReturn([$quoteItem]);
 
-        $total = $this->createPartialMock(
-            Total::class,
-            ['setBaseVirtualAmount', 'setVirtualAmount']
-        );
+        $total = $this->getMockBuilder(Total::class)
+            ->disableOriginalConstructor()
+            ->disableOriginalClone()
+            ->disableArgumentCloning()
+            ->disallowMockingUnknownTypes()
```

Comment:
**YAGNI** If original code is not performing any `clone` then you don't need to provide these methods.